include/wx/private/spinctrl.h

```diff
@@ -26,6 +26,8 @@ extern int GetMaxValueLength(int minVal, int maxVal, int base);
 // of wxSpinCtrl is derived from wxSpinButton but uses the same algorithm.
 extern wxSize GetBestSize(const wxControl* spin, int minVal, int maxVal, int base);
 
+// Helper function to check if given combination of control parameters is valid.
+extern bool AreParametersValid(int minVal, int maxVal, int base);
```

Comment:
Maybe we should rename it to `IsBaseCompatibleWithRange()` or something like that? Then we could save on all the comments currently used before every call to it.

Reply:
Yes, I agree with something similar to `IsBaseCompatibleWithRange()`.
I also agree this patch is the better approach than [#1939](https://github.com/wxWidgets/wxWidgets/pull/1939)

Reply:
Indeed `IsBaseCompatibleWithRange` is a better name.